packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -170,7 +169,10 @@ export default class Dropdown extends React.Component<DropdownProps> {
                     }
                     modifiers={{
                         wbVisibility: visibilityModifierDefaultConfig,
-                        preventOverflow: {boundariesElement: "viewport"},
+                        preventOverflow: {
+                            boundariesElement: "viewport",
+                            escapeWithReference: true,
```

Comment:
What does this do?

Reply:
It keeps the anchor and popper element in the same relative position with each other. There's still some kinks in the popper positioning I'm looking at.